timesync/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+var (
+	Values = DefaultConfig()
```

Comment:
If we won't set this to timeconfig in `main.p2p` it will never be loaded. please take care of this.